sycl/include/CL/sycl/handler.hpp

```diff
@@ -462,16 +462,16 @@ class __SYCL_EXPORT handler {
   }
 
   static id<2> getDelinearizedIndex(const range<2> Range, const size_t Index) {
-    size_t x = Index / Range[1];
-    size_t y = Index % Range[1];
-    return {x, y};
+    size_t x = Index % Range[1];
```

Comment:
Could you please clarify what modifications in this file change?

Reply:
Functionally, there is no change. But the original code was wrong in that x is not  Index divided by y-term, it's Index modulo y-term.   And y was reversed as well.   And now the the return value agrees with how ranges are used in buffers, etc.